ballerina/modules/apex/client.bal

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Check for other places as well.
```suggestion
// Copyright (c) 2024 WSO2 LLC. (http://www.wso2.org).
```

Reply:
Hey why is this wrong, I checked other modules also

Reply:
https://github.com/niveathika/module-ballerinax-sap.s4hana.procurement/blob/9677c6118942096e6aa12c4f2068350fa873f4de/docs/sanitation/sanitations.bal

Reply:
Pls check this email [1]

[1] - `Re: License Header for WSO2 code`